### PR TITLE
refs_5196_show_data

### PR DESCRIPTION
**Pull request by tungct97**
https://edu-redmine.framgia.vn/issues/5196

### CODE REVIEW

**Comment by tungct97** [comment]
ready

File: app/src/main/java/com/framgia/music_51/data/repository/TrackRepository.java [hunk 1]
```diff
@@ -36,6 +37,7 @@ public static TrackRepository getInstance(TrackDataSource.Remote remote, TrackDa
 
     @Override
     public void addFavorite(Track track) {
+        Log.d("TAG1", "addFavorite: "+track.isFavourite());
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/music_51/data/repository/TrackRepository.java]: remove log

File: app/src/main/res/layout/fragment_person.xml [hunk 2]
```diff
@@ -1,9 +1,57 @@
...
+            android:text="@string/title_download"
+            android:textSize="@dimen/sp_20"
+            android:textStyle="bold"
+            app:layoutManager="android.support.v7.widget.LinearLayoutManager"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/recycler_favourite" />
+
+        <android.support.v7.widget.RecyclerView
+            android:id="@+id/recycler_download"
+            android:layout_width="match_parent"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_person.xml]: Trong ConstraintLayout view con không khai báo là match_parent

**Comment by tungct97** [comment]
ready

**Pull request merged**